README.md

```diff
@@ -2,7 +2,7 @@
 
 NASA’s [Open-Source Science Initiative \(OSSI\)](https://science.nasa.gov/open-science-overview) is a comprehensive program of activities to enable and support moving science towards openness, and it aims to implement [SMD's Strategy for Data Management and Computing for Groundbreaking Science 2019-2024](https://science.nasa.gov/files/science-red/s3fs-public/atoms/files/SDMWG%20Strategy_Final.pdf). OSSI includes a commitment to the open sharing of software, data, and knowledge (algorithms, papers, documents, ancillary information) as early as possible in the scientific process. The principles of open-source science are to make publicly funded scientific research transparent, inclusive, accessible, and reproducible. 
 
-As part of the OSSI, [SMD’s Scientific Information Policy](https://science.nasa.gov/researchers/science-data/science-information-policy) provides guidance on the open sharing of publications, data, and software created in the pursuit of scientific knowledge. [SPD-41a: Scientific Information Policy for the Science Mission Directorate](https://science.nasa.gov/science-red/s3fs-public/atoms/files/SMD-information-policy-SPD-41a.pdf) updates the previously released [SPD-41](https://science.nasa.gov/science-red/s3fs-public/atoms/files/Scientific%20Information%20policy%20SPD-41.pdf), which consolidated existing Federal and NASA policy on sharing scientific information.  SPD-41a was developed in light of new federal guidance on [Ensuring Free, Immediate, and Equitable Access to Federally Funded Research](https://www.whitehouse.gov/wp-content/uploads/2022/08/08-2022-OSTP-Public-Access-Memo.pdf), studies from the National Academies, and input from the SMD scientific community. 
+As part of the OSSI, [SMD’s Scientific Information Policy](https://science.nasa.gov/researchers/science-data/science-information-policy) provides guidance on the open sharing of publications, data, and software created in the pursuit of scientific knowledge. [SPD-41a: Scientific Information Policy for the Science Mission Directorate](https://science.nasa.gov/science-red/s3fs-public/atoms/files/SMD-information-policy-SPD-41a.pdf) updates the previously released [SPD-41](https://smd-cms.nasa.gov/wp-content/uploads/2023/05/ScientificInformationpolicySPD_41.pdf), which consolidated existing Federal and NASA policy on sharing scientific information.  SPD-41a was developed in light of new federal guidance on [Ensuring Free, Immediate, and Equitable Access to Federally Funded Research](https://www.whitehouse.gov/wp-content/uploads/2022/08/08-2022-OSTP-Public-Access-Memo.pdf), studies from the National Academies, and input from the SMD scientific community. 
```

Comment:
Change the link for SPD-41a to "https://science.nasa.gov/spd-41/"  (it is still pointing to the old link)